ansible/dimensiondata/dimensiondata_backup.py

```diff
@@ -38,8 +38,11 @@
   region:
     description:
       - The target region.
-    choices: %s
-    default: na
+    choices:
+      - Regions are defined in Apache libcloud project file = libcloud/common/dimensiondata.py 
+      - See https://libcloud.readthedocs.io/en/latest/compute/drivers/dimensiondata.html
+      - Note: Values avail in array dd_regions(). 
+    default: na aka North America
```

Comment:
na seems to me be a very confusing value for a default value.

Makes is seem like NOT APPLICABLE.

Which is why I got more specific

On Tue, Jul 12, 2016 at 11:14 AM, Aimon Bustardo <notifications@github.com>
wrote:

> In ansible/dimensiondata/dimensiondata_backup.py
> <https://github.com/NexusIS/ansible-dimensiondata/pull/1#discussion_r70491485>
> :
>
> > @@ -38,8 +38,11 @@
> >    region:
> >      description:
> >        - The target region.
> > -    choices: %s
> > -    default: na
> > +    choices:
> > +      - Regions are defined in Apache libcloud project file = libcloud/common/dimensiondata.py
> > +      - See https://libcloud.readthedocs.io/en/latest/compute/drivers/dimensiondata.html
> > +      - Note: Values avail in array dd_regions().
> > +    default: na aka North America
>
> This can confuse the reader on what the actual default value is. Please
> remove here and in other places. They can read link you added for more
> info.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NexusIS/ansible-dimensiondata/pull/1/files/8df60955ffdbeda67007f8a82b7dcc4e23ee7d17#r70491485>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AI6_IeSZFnwZSK-__N6tVU3bRiVTrKR9ks5qU9l2gaJpZM4JKqRJ>
> .
>